apps/runplus/app.js

```diff
@@ -49,9 +54,15 @@ let statIDs = [settings.B1,settings.B2,settings.B3,settings.B4,settings.B5,setti
 let exs = ExStats.getStats(statIDs, settings);
 // ---------------------------
 
+function inPauseWindow() {
+  return exs.state.duration <= settings.resume.promptAfter;
+}
+
 function setStatus(running) {
-  layout.button.label = running ? "STOP" : "START";
-  layout.status.label = running ? "RUN" : "STOP";
+  const paused = isInPauseWindow();
```

Comment:
... this don't match :)

Reply:
That's what I get for last minute changes! Thank you